src/executor/executor.ts

```diff
@@ -101,6 +103,12 @@ export type YagnaOptions = {
  * A high-level module for defining and executing tasks in the golem network
  */
 export class TaskExecutor {
+  /**
+   * EventEmitter (EventEmitter3) instance emitting TaskExecutor events.
+   * @see TaskExecutorEventsDict for available events.
+   */
+  readonly events: EventEmitter<TaskExecutorEventsDict> = new EventEmitter();
```

Comment:
I'm not sure if that was hinted in any of the commits: this kind of a change is a `BREAKING CHANGE` so please make sure that you use that statement right so that `semantic-release` can pick this up correctly.